app/src/substrate/substrate_functions_V19.h

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ *  (c) 2019 - 2023 Zondax AG
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ********************************************************************************/
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "parser_common.h"
+#include "substrate_methods_V19.h"
```

Comment:
Probably we can remove this file or add a comment if removing a header causes too much pain.

// Versioned types for Tx version:19